codemp/server/sv_main.cpp

```diff
@@ -598,8 +598,7 @@ void SVC_Info( netadr_t from ) {
 	Info_SetValueForKey( infostring, "mapname", sv_mapname->string );
 	Info_SetValueForKey( infostring, "clients", va("%i", count) );
 	Info_SetValueForKey( infostring, "g_humanplayers", va("%i", humans) );
-	Info_SetValueForKey( infostring, "sv_maxclients",
-		va("%i", sv_maxclients->integer - sv_privateClients->integer ) );
+	Info_SetValueForKey( infostring, "sv_maxclients", va("%i", sv_maxclients->integer ) );
```

Comment:
Well it is still ambiguous, for example if there are 12 clients and 2 private clients, server would show 14/14 even though there is still space. This should stay a client side fix imo (14/14+2).